include/irr/asset/ICPUVirtualTexture.h

```diff
@@ -307,9 +318,8 @@ class ICPUVirtualTexture final : public IVirtualTexture<ICPUImageView, ICPUSampl
         reinterpret_cast<SViewAliasTextureData*>(&aliasAddr)[0];
     }
 
-    bool free(const STextureData& _addr) override
+    bool free(const SMasterTextureData& _addr) override
```

Comment:
and where is the separate free/destroy for the aliases?

Reply:
in IvirtualTexture